### PR TITLE
Add ability to set more env vars via config & change env template & f…

### PR DESCRIPTION
**Pull request by th30nlyw4y**
**What this PR does**:
- Adds ability to provide Twilio and two more Slack env values through helm values
- Adds ability to specify env vars by array (for backwards compatibility) or by map
- Fixes spacing of env vars (in engine & celery deployment -- containers + initContainers):
    before:
    ```yaml
    env:
                - name: BASE_URL
                   value: https://example.com
    ```
    after:
    ```yaml
    env:
        - name: BASE_URL
           value: https://example.com
    ```
**Which issue(s) this PR fixes**: #518

### CODE REVIEW

**Comment by th30nlyw4y** [comment]
Done rebasing, should be fine now

**Comment by th30nlyw4y** [comment]
Ok, let's add it. I see no problems here

**Comment by th30nlyw4y** [comment]
HI, @iskhakov !
Sorry for bothering you, but could you, please, check this PR ?
I think these improvements would really simplify env vars configuration

**Review by iskhakov — APPROVED** [review]
Looks good, thank you for the contribution! 

One small note, these values are going to be applied only to the new installations, they will not work for the "upgraded" installations, because live settings (same variables stored in the database and available from web interface on `Env Variables` page) are already populated. We need to change that logic and prefer env variables if the live settings are set to default values.

Approving because it doesn't break anything

File: helm/oncall/values.yaml [hunk 1]
```diff
@@ -96,13 +98,27 @@ oncall:
     password: ~
     tls: ~
     fromEmail: ~
+  twilio:
+    # Twilio account SID/username to allow OnCall to send SMSes and make phone calls
+    accountSid: ~
+    # Twilio password to allow OnCall to send SMSes and make calls
+    authToken: ~
+    # Number from which you will receive calls and SMS
+    phoneNumber: ~
```
Comment (iskhakov) [inline helm/oncall/values.yaml:107]: Please specify that this _value must be in quotes_ otherwise it has int type and rendered as ` 1.234567e+10`

**Review by iskhakov — CHANGES_REQUESTED** [review]

File: helm/oncall/values.yaml (hunk 1 above)
Comment (th30nlyw4y) [reply helm/oncall/values.yaml:107]: Sorry, forgot about that thing! (We've faced the same issue :))

**Review by iskhakov — APPROVED** [review]

**Comment by th30nlyw4y** [comment]
@iskhakov sorry for bothering you again (i hope that's the last time :)), but should i wait while someone merges this or do smth else by myself ? As i can see, a can't merge it even after all checks (and i'm not sure that i should be able to do that)

**Comment by th30nlyw4y** [comment]
Hi, @Matvey-Kuk !
Maybe you could help here ?)

**Comment by iskhakov** [comment]
Thank you for the contribution. Merging and releasing shortly

**Pull request merged**